testsuite/openshift/__init__.py

```diff
@@ -52,3 +54,28 @@ def _wrap(self, *args, **kwargs):
             func(self, *args, **kwargs)
 
     return _wrap
+
+
+@dataclass
+class MatchExpression:
+    """
+    Data class intended for defining K8 Label Selector expressions.
+    Used by selector.matchExpressions API key identity.
+    """
+
+    operator: Literal["In", "NotIn", "Exists", "DoesNotExist"]
+    values: list[str]
+    key: str = "group"
+
+
+@dataclass
+class Selector:
```

Comment:
Any reason you moved these dataclasses here? `policy.authorization` seemed like a good place

Reply:
Yes, these classes are from kubernetes not from any policy in particular